games/devtest/mods/unittests/misc.lua

```diff
@@ -6,6 +6,21 @@ local function test_random()
 end
 unittests.register("test_random", test_random)
 
+local function test_cxx_exceptions()
+	local ok, msg = pcall(core.get_content_id, "~!~")
+	assert(not ok)
+	assert(type(msg) == "string")
+	assert(msg:find("~!~", 1, true) ~= nil)
+end
+unittests.register("test_cxx_exceptions", test_cxx_exceptions)
+
+local function test_lua_exceptions()
+	local ok, msg = pcall(error, "error")
+	assert(not ok)
+	assert(msg == "error")
+end
```

Comment:
There's no reason this wouldn't work, is there?

Reply:
You're probably right.